assets/css/components/_segmented-control.scss

```diff
@@ -0,0 +1,26 @@
+.segmented-control {
```

Comment:
I see that "segmented-control" is a term that was already being used, but it feels a bit clunky to me. `tabs` is more descriptive IMO

Reply:
Yep, I could go either way.

I believe the term “Segmented Control” comes from [Apple’s HIG](https://developer.apple.com/design/human-interface-guidelines/ios/controls/segmented-controls/). Perhaps that’s why it was used here?

Reply:
Frankly, I want to remove this entire bar eventually…I think there are better design patterns we could leverage to filter the messages (which is really all this is doing, plus a link to Interests).

Reply:
Ooh yeah that makes sense about Apple. I had a similar thought when I went to look at the live site. There's a lot of blank space not being used 🤠 

I'm not super opinionated about this, so if you leave it as `segmented-control` I won't fight you.

Reply:
![](https://media.giphy.com/media/1RzxeL2PuHYD1pw32i/giphy.gif)